### PR TITLE
feat: add GA4 tracking

### PR DESCRIPTION
**Pull request by JerryY999**

### CODE REVIEW

File: packages/ui/src/navigation/NavigationLite.tsx [hunk 1]
```diff
@@ -31,6 +32,18 @@ export const NavigationLite = ({
...
+  useEffect(() => {
+    try {
+      if (typeof window !== 'undefined') {
+        const host = window?.location.host
+        setIsTestnet(host.indexOf('testnet') > -1)
+      }
+    } catch (e) {
+      console.log(e)
+    }
+  }, [])
```
Comment (wmzy) [inline packages/ui/src/navigation/NavigationLite.tsx:45]: There is no need for a react state, just something like this:
```ts
const isTestnet = typeof window !== 'undefined' && window.location.host.includes('testnet');
```

And is there any other way to tell apart from hard-coding?

Comment (JerryY999) [reply packages/ui/src/navigation/NavigationLite.tsx:45]: 有环境变量配置,我没引进来

Comment (grezle) [reply packages/ui/src/navigation/NavigationLite.tsx:45]: So why not use the env? All of the preview environments are set to use testnet by default.

**Review by grezle — COMMENTED** [review]
Are you deliberately only adding analytics to the bridge and faucet? You have modified navigation-lite, is that not being used by the migrator/converter as well?

**Review by wmzy — APPROVED** [review]

File: packages/ui/src/navigation/NavigationLite.tsx (hunk 1 above)
Comment (JerryY999) [reply packages/ui/src/navigation/NavigationLite.tsx]: i have fixed this issue

**Pull request merged**